src/controllers/commentController.ts

```diff
@@ -0,0 +1,41 @@
+import { type Context, type Env } from 'hono';
+import { type CommentContext, type IdContext } from '../models/contextModel';
+import { commentRepository } from '../dataAcces/commentRepository';
+
+export const commentController = {
+  async findAll(
+    c: Context<Env, '/comments', Record<string, unknown>>
+  ): Promise<Response> {
+    const comments = await commentRepository.getAll();
+
+    return c.json({ data: comments, ok: true }, 200);
+  },
+
+  async findById(c: IdContext): Promise<Response> {
+    const { id } = c.req.valid('param');
```

Comment:
i suggest rename this variable c -> context

Reply:
If we need more meaningful name, I'll rename it. But right now the whole app uses "c", I named it that way because the hono documentation uses that name.